utils/config.py

```diff
@@ -34,6 +35,11 @@ def relative_path(script_reference_path, rel_path):
         if zone_name_1 not in ZONE_NEIGHBOURS:
             ZONE_NEIGHBOURS[zone_name_1] = set()
         ZONE_NEIGHBOURS[zone_name_1].add(zone_name_2)
+        if (zone_name_1 not in INTERCONNECTORS) and ('capacity' in v):
+            INTERCONNECTORS[zone_name_1] = {}
+        INTERCONNECTORS[zone_name_1][zone_name_2] = {
```

Comment:
isn't that redundant with EXCHANGES_CONFIG[f'{zone_name1}->{zone_name2}'] ?

Reply:
🤦 Yes indeed that makes everything simpler